src/View/Helper/FormHelper.php

```diff
@@ -1008,7 +1008,8 @@ public function input($fieldName, array $options = [])
             'required' => null,
             'options' => null,
             'templates' => [],
-            'templateVars' => []
+            'templateVars' => [],
+            'optionsLabel' => true
```

Comment:
Because the labels show as default, you can now pass `false` to skip the labels wrapping radio/checkboxes